### PR TITLE
RUM & Logs package bump

### PR DESCRIPTION
**Pull request by bgdeutsch**
### What does this PR do?
- bumps package version for DD Rum & Logs
- adds IA subdomain param to config
- updates config so data populates in the `Documentation - Prod` RUM application in DD

### Motivation
https://datadoghq.atlassian.net/browse/WEB-2989

### Preview
https://docs-staging.datadoghq.com/brian.deutsch/rum-bump

- [ ] RUM data is populating correctly ([my preview session here](https://dd-corpsite.datadoghq.com/rum/explorer?query=%40application.id%3A3493b4e7-ab12-4852-8836-ba96af7bc745%20%40type%3Asession%20env%3Apreview&cols=&event=AgAAAYWXcsrRjyubmQAAAAAAAAAYAAAAAEFZV1hjc3JSQUFDVzRaXzRxaEV4REFHQQAAACQAAAAAMDE4NTk3ODYtNzRkNS00M2MwLWExMmEtMDMwYTg5NWQ2OGM3&from_ts=1673198704034&to_ts=1673285104034&live=true))
- [ ] RUM data surfaces in the `Documentation - Prod` application in DD under `preview` env.
- [ ] frustration signals still work
- [ ] confirm RUM SDK requests are using the `iam-rum-intake` subdomain (in browser tools navigate to `Network`, filter results by `Fetch/XHR`, inspect headers.   This should work in Chrome and FF.)

### Additional Notes
We'll remove the `Documentation - Staging` app in DD once the current data is no longer retained
---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[WEB-2989]: https://datadoghq.atlassian.net/browse/WEB-2989?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nsollecito** [comment]
@bgdeutsch as part of these changes, can we combine the Docs Staging and Prod application config so that we are reporting as part of the same RUM app? Currently we have 2 separate instances where we should opt to have a single application with the proper env config set for `staging` v `live`

cc @apigirl FYI -- let me know if you have any concerns.

**Review by apigirl — APPROVED** [review]

**Comment by apigirl** [comment]
hey @bgdeutsch and @nsollecito, I don't actually think we need rum on the staging site. It looks like we don't use it (we are using the production data though)

**Comment by nsollecito** [comment]
@apigirl thanks, I would like to keep it enabled though, it comes in handy for debugging/troubleshooting. But we should clean it up and make sure we're setting the proper config

File: assets/scripts/components/dd-browser-logs-rum.js [hunk 1]
```diff
@@ -22,33 +25,41 @@ if (window.DD_RUM) {
             version: CI_COMMIT_SHORT_SHA,
             trackInteractions: true,
             trackFrustrations: true,
-            enableExperimentalFeatures: ["frustration-signals","clickmap"],
+            enableExperimentalFeatures: ["clickmap"],
             sampleRate: 50,
-            premiumSampleRate: 50,
-            allowedTracingOrigins: [window.location.origin]
+            sessionReplaySampleRate: 50,
```
Comment (bgdeutsch) [inline assets/scripts/components/dd-browser-logs-rum.js:30]: https://docs.datadoghq.com/real_user_monitoring/browser/#configuration

File: assets/scripts/config/config-docs.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ module.exports = {
         gaTag: 'UA-21102638-5'
     },
     preview: {
-        ddClientToken: 'pub36877d3864fab670b5ae7e1d5d26cb08',
-        ddApplicationId: 'c4e83ad8-4eda-4d2e-aae1-d943abce0463',
+        ddClientToken: 'pub16bb5ef3e9bf55f156338987e27246c7',
+        ddApplicationId: '3493b4e7-ab12-4852-8836-ba96af7bc745',
```
Comment (bgdeutsch) [inline assets/scripts/config/config-docs.js:17]: Consolidating docs RUM applications

**Review by nsollecito — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**